### PR TITLE
updated more courses under devops sec.

### PR DESCRIPTION
**Pull request by AshishSharma1203**
## What does this PR do?
Add resources  under DevOps section regarding issue #8440

### Why is this valuable (or not)?
Because it closes issue #8440 

### How do we know it's really free?
Its all available on youtube for free. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -479,6 +479,11 @@
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
 * [DevOps Tutorial for Beginners](https://www.youtube.com/playlist?list=PLVHgQku8Z934suC9LSE6vaAKjOH_MfRbE) - Intellipaat
 * [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Karl Krueger, Dwayne Lessner, Gundega Dekena (Udacity)
+* [Simplilearn 9 hours DevOps Course](https://www.youtube.com/watch?v=lpk7VpGqkKw) <br/>
```
Comment (shrn01) [inline courses/free-courses-en.md:482]:
```suggestion
* [Simplilearn 9 hours DevOps Course](https://www.youtube.com/watch?v=lpk7VpGqkKw) - Simplilearn
```
Please remove the `<br/>` and add channel attributions for all these entries

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (AshishSharma1203) [reply courses/free-courses-en.md]: made the required changes . please verify it now .

**Review by shrn01 — COMMENTED** [review]
Aphabetically sort the resources

**Comment by AshishSharma1203** [comment]
> Aphabetically sort the resources

done

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -476,9 +476,15 @@
 ### DevOps
 
 * [AWS DevOps Engineer Learning Plan](https://explore.skillbuilder.aws/learn/public/learning_plan/view/85/devops-engineering-learning-plan?la=sec&sec=lp) - AWS
+* [Computer Networking 4 hours Course](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
 * [DevOps Tutorial for Beginners](https://www.youtube.com/playlist?list=PLVHgQku8Z934suC9LSE6vaAKjOH_MfRbE) - Intellipaat
+* [freeCodeCamp.org 2 hours DevOps Engineering Course](https://www.youtube.com/watch?v=j5Zsa_eOXeY) -FreeCodeCamp
 * [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Karl Krueger, Dwayne Lessner, Gundega Dekena (Udacity)
+* [Learn DevOps playlist by Cloud Advocate](https://www.youtube.com/playlist?list=PLBRBRV08tHh1W6F9ijILxGJZNfzFuytZo) - Cloud Advocate
+* [Simplilearn 9 hours DevOps Course](https://www.youtube.com/watch?v=lpk7VpGqkKw) -Simplilearn
```
Comment (shrn01) [inline courses/free-courses-en.md:485]:
```suggestion
* [Simplilearn 9 hours DevOps Course](https://www.youtube.com/watch?v=lpk7VpGqkKw) - Simplilearn
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -476,9 +476,15 @@
...
+* [Tech World with Nana's Youtube channel](https://www.youtube.com/c/TechWorldwithNana) -Tech World With nana
```
Comment (shrn01) [inline courses/free-courses-en.md:486]: Please add a specific playlist or a video greater than 1 hr in length

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -476,9 +476,15 @@
 ### DevOps
 
 * [AWS DevOps Engineer Learning Plan](https://explore.skillbuilder.aws/learn/public/learning_plan/view/85/devops-engineering-learning-plan?la=sec&sec=lp) - AWS
+* [Computer Networking 4 hours Course](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
 * [DevOps Tutorial for Beginners](https://www.youtube.com/playlist?list=PLVHgQku8Z934suC9LSE6vaAKjOH_MfRbE) - Intellipaat
+* [freeCodeCamp.org 2 hours DevOps Engineering Course](https://www.youtube.com/watch?v=j5Zsa_eOXeY) -FreeCodeCamp
```
Comment (shrn01) [inline courses/free-courses-en.md:482]:
```suggestion
* [freeCodeCamp.org 2 hours DevOps Engineering Course](https://www.youtube.com/watch?v=j5Zsa_eOXeY) - FreeCodeCamp
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 3 above)
Comment (AshishSharma1203) [reply courses/free-courses-en.md]: done . can u verify it now.

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -476,9 +476,11 @@
 ### DevOps
 
 * [AWS DevOps Engineer Learning Plan](https://explore.skillbuilder.aws/learn/public/learning_plan/view/85/devops-engineering-learning-plan?la=sec&sec=lp) - AWS
+* [Computer Networking 4 hours Course](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```
Comment (eshellman) [inline courses/free-courses-en.md:479]: This should be in Networking

Comment (AshishSharma1203) [reply courses/free-courses-en.md]: removed that one .

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -479,6 +479,7 @@
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
 * [DevOps Tutorial for Beginners](https://www.youtube.com/playlist?list=PLVHgQku8Z934suC9LSE6vaAKjOH_MfRbE) - Intellipaat
 * [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Karl Krueger, Dwayne Lessner, Gundega Dekena (Udacity)
+* [Simplilearn 9 hours DevOps Course](https://www.youtube.com/watch?v=lpk7VpGqkKw) - Simplilearn
```
Comment (shrn01) [inline courses/free-courses-en.md:482]:
```suggestion
* [DevOps Full Course](https://www.youtube.com/watch?v=lpk7VpGqkKw) - Simplilearn
```

Dont change the resource name

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 6 above)
Comment (eshellman) [reply courses/free-courses-en.md:482]: Don't invent titles! Use the title given by the resource.

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -479,6 +479,7 @@
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
 * [DevOps Tutorial for Beginners](https://www.youtube.com/playlist?list=PLVHgQku8Z934suC9LSE6vaAKjOH_MfRbE) - Intellipaat
 * [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Karl Krueger, Dwayne Lessner, Gundega Dekena (Udacity)
+* [DevOps Full Course ](https://www.youtube.com/watch?v=lpk7VpGqkKw) - Simplilearn
```
Comment (shrn01) [inline courses/free-courses-en.md:482]: Order the resources in alphabetical order. It should fix the lint errors

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**